### DISCUSSION TITLE
Link/Entry fieldtype - Ability to select child elements of the chosen collections only

### DISCUSSION THREAD

**Discussion by CedsTrash** [statamic/cms#5890] [Q&A]
Hi, I was wondering if there was a way to only display child elements when selecting an entry.

I have a "Pages" collection, and I added an Link field to the blueprint, in order to attach another page. But I don't want my content editors to be able to select a parent level page, just child pages.
Same thing for Entries.

Is there a way to do this?

Thanks a lot for your insights!

**Comment by afonic** [comment]
I don't think that there is a build-in way.

In order to do this, you can probably create a custom fieldtype, based on the `Link` fieldtype and change the entries it pull in the `preload` method.

**Reply by CedsTrash** [reply]
Thanks a lot for pointing me towards this. I started to investigate but I'd have to extend/override many nested Vue components and create a new "CP" endpoint in order to run a custom query to return only child entries. Too much work compared to the small benefit :-)

Thanks again.
